frontend/tailwind.config.js

```diff
@@ -1,12 +1,58 @@
 /** @type {import('tailwindcss').Config} */
+
 module.exports = {
-  purge: ['./**/*.html', './**/*.js',],
-  content: ["*/templates/**/*.html"],
+  future: {},
+  content: ["../*/templates/*.{html,js}", "../*/templates/**/*.{html,js}"],
+  darkMode: "media",
   theme: {
-    extend: {},
+    extend: {
+      screens: {
+          "nav-breakpoint": "1050px", // prevents nav  links from wrapping into two lines
```

Comment:
Could we use `lg:` instead? It's pretty close to the default tailwind `1024px` breakpoint :)